apps/src/tutorialExplorer/filterGroupContainer.js

```diff
@@ -0,0 +1,44 @@
+/* FilterGroupContainer: The container UI for a group of filter choices.
+ * The choices themselves should be passed in as children.
+ */
+
+import React, {PropTypes} from 'react';
+import { getResponsiveValue } from './responsive';
+
+const styles = {
+  filterGroupOuter: {
+    float: "left",
+    paddingBottom: 20,
+    paddingRight: 40,
+    paddingLeft: 10
+  },
+  filterGroupText: {
+    fontFamily: '"Gotham 5r", sans-serif',
+    borderBottom: 'solid grey 1px'
+  }
+};
+
+const FilterGroupContainer = React.createClass({
```

Comment:
In anticipation of upcoming changes to React ES6 that will no longer support .createClass, we want to use `const class FilterGroupContainer extends Component` and import `Component` like `PropTypes` at the top of the file.  You can see an example here: https://github.com/code-dot-org/code-dot-org/pull/18194

Reply:
...good point.  I'd like to make this change in a separate PR for all of tutorialExplorer at once.